tests/acceptance/features/apiActivities/activities.feature

```diff
@@ -35,11 +35,11 @@ Feature: check activities
                   "required": ["message","variables"],
                   "properties": {
                     "message": {
-                      "const": "{user} added {resource} to {space}"
+                      "const": "{user} added {resource} to {folder}"
                     },
                     "variables": {
                       "type": "object",
-                      "required": ["resource","space","user"],
+                      "required": ["resource","folder","user"],
```

Comment:
only the message template is change, so the property name will still be `space` not a `folder`
```json
"variables": {
          "resource": {
            "id": "cb2e3f6d-4e13-42de-ba61-4a1b7ff6f374$2503e6a8-54fb-4e0f-b002-e43c90a15b35!e0800c08-092f-41e7-a5a3-332b1d6b7532",
            "name": "lorem.txt"
          },
          "space": {
            "id": "2503e6a8-54fb-4e0f-b002-e43c90a15b35!2503e6a8-54fb-4e0f-b002-e43c90a15b35",
            "name": "Admin"
          },
          "user": {
            "id": "2503e6a8-54fb-4e0f-b002-e43c90a15b35",
            "displayName": "admin"
          }
        }
```
```suggestion
                      "required": ["resource","space","user"],
```

Reply:
`"required": [` field should not be changed